static/js/emoji_picker.js

```diff
@@ -390,6 +398,14 @@ exports.navigate = function (event_name) {
         return;
     }
 
+    if (event_name === 'enter') {
+        if (is_composition(e.target)) {
+            e.target.click();
+        } else {
+            toggle_selected_emoji(e);
+        }
```

Comment:
This should return true, right?  The interface for this function is the same as all the hotkey.js ones -- true means we processed it and the caller should do `preventDefault` + `stopPropagation`.

And I think the new caller should pass on the return value from this rather than always returning true.

(I also notice the `return`  above is probably wrong?).

Reply:
Yes, I amended the `Move enter handling into picker.` commit.